### DISCUSSION TITLE
Lunches data

### DISCUSSION THREAD

**Discussion by mgrfilipmarek** [rine77/homeassistantedupage#66] [Ideas]
Any idea how to present lunches data in HA? Is calendar entity the best option?

Currently API offers only lunches (other meals of the day are not provided). Possibly other meals of the day will be added into API.
There is a lot of information we can load. Menu, allergens, rating, weight, served_time.... Which one you would like to see available in HA? 
Documentation: https://edupageapi.github.io/edupage-api/lunches.html#edupage_api.lunches.Lunch

**Comment by rine77** [comment]
> Any idea how to present lunches data in HA? Is calendar entity the best option?
> 
> Currently API offers only lunches (other meals of the day are not provided). Possibly other meals of the day will be added into API. There is a lot of information we can load. Menu, allergens, rating, weight, served_time.... Which one you would like to see available in HA? Documentation: https://edupageapi.github.io/edupage-api/lunches.html#edupage_api.lunches.Lunch

I think a separate lunch calendar would be best, so you can use it or not depending on you school!
